### ISSUE TITLE
Possible? Keep item renderers in virtual layout after first creation.

### ISSUE THREAD

**Issue by harry248** [BowlerHatLLC/feathers#450]
Is it possible to keep the item renderers in a list with a virtual layout as soon as they have been created? I have a list with lots of ImageLoader instances, so using useVirtualLayout=false is not really an option. But with a virtual layout it also does not feel very nice because the images are always loaded again. It would be nice if the item renderers are still being created deferred the first time, however then are kept persistent. Some kind of itemCreationPolicy...

**Comment by kevinfoley** [comment]
@joshtynjala  Just stumbled on this old issue while looking for something else; maybe this issue is no longer relevant since you have the TextureCache for ImageLoaders?

**Comment by joshtynjala** [comment]
Indeed, for the specific issue of reloading images, the TextureCache should do the trick, at least up to the maximum cache size. I can imagine there might be other reasons why someone might want to keep using the same item renderer for a particular item. Maybe there are other things in a custom item renderer that need to be restored and don't have a cache.

An interesting hack would be to use factoryIDFunction and create a separate factory for each item in the data provider. Each factory would always return the same item renderer, which you could probably store in an Array or Dictionary.
